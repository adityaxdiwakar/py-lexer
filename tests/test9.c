int main() {
    int number = 10;

    if (number > 5) {
        printf("The number is greater than 5.\n");
    } else {
        printf("The number is not greater than 5.\n");
    }

    return 0;
}