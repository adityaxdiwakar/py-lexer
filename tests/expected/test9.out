INT
MAIN
LPAREN
RPAREN
LBRACK
INT
IDENT(number)
EQUALS
INT(10)
SEMI
IF
LPAREN
IDENT(number)
GT
INT(5)
RPAREN
LBRACK
IDENT(printf)
LPAREN
STR("The number is greater than 5.\n")
RPAREN
SEMI
RBRACK
ELSE
LBRACK
IDENT(printf)
LPAREN
STR("The number is not greater than 5.\n")
RPAREN
SEMI
RBRACK
RETURN
INT(0)
SEMI
RBRACK